pkg/assert/node.go

```diff
@@ -21,7 +21,7 @@ func NodeAssertVersionTypeUpgrade(c customflag.FlagConfig) NodeAssertFunc {
 		return assertCommit(c)
 	}
 
-	return func(g Gomega, node shared.Node) {
+	return func(_ Gomega, _ shared.Node) {
```

Comment:
why this change?

Reply:
As suggested by linter
```
```

Reply:
ok , so lets remove `	return func(Gomega, shared.Node) {` , not ommit _ 

_  is supposed to be used when u need a parameter on the signature but its not going to use it inside.

is slightly different from what linter said

Reply:
Ok added it back and not seeing ci-linter complaining 🤷